tests/containers/setup_centos.pm

```diff
@@ -0,0 +1,26 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Setup Centos
+# - setup networking via dhclient
+# - import SUSE CA certificates
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use Mojo::Base qw(consoletest);
+use testapi;
+
+sub run {
+    my ($self) = @_;
+    $self->select_serial_terminal;
+    assert_script_run "dhclient -v";
```

Comment:
I think we can come up with image which would try to do this automatically .

Reply:
After discussion with @asmorodskyi i resolve this discussion as it is easier to overcome the problem with the different NICs using dhclient